src/pages/besquare/index.js

```diff
@@ -12,14 +12,22 @@ import WhyDeriv from './components/sections/_why-deriv'
 import Layout from 'components/layout/layout'
 import { SEO } from 'components/containers'
 import { WithIntl, localize } from 'components/localization'
+import BeSquareOg from 'images/common/be-square/besquare-og.png'
+
+const meta_attributes = {
+    og_title: localize('BeSquare graduate programme by Deriv'),
```

Comment:
I think we should remove the `localize` in here as well based on https://github.com/binary-com/deriv-com/pull/1507/files